crates/dojo-lang/src/plugin_test_data/inline_macros

```diff
@@ -191,15 +191,17 @@ fn foo() {
     {
         let mut keys = Default::<array::Array>::default();
         let mut data = Default::<array::Array>::default();
-        starknet::Event::append_keys_and_data(@Struct { x: 10,  }, ref keys, ref data);
+        starknet::Event::append_keys_and_data(
+            @Into::<_, Event>::into(Struct { x: 10,  }), ref keys, ref data
+        );
         world.emit(keys, data.span());
     };
 
     let id = (0x420, 0x1337);
     {
         let mut keys = Default::<array::Array>::default();
         let mut data = Default::<array::Array>::default();
-        starknet::Event::append_keys_and_data(@id, ref keys, ref data);
+        starknet::Event::append_keys_and_data(@Into::<_, Event>::into(id), ref keys, ref data);
         world.emit(keys, data.span());
     };
```

Comment:
should this be removed? because now `emit!` only makes sense on `Event`, this will confuse contributors reading through the test.